### PR TITLE
Add Domain Transfer Examples

### PR DESCRIPTION
**Pull request by duduribeiro**
This commit adds example for
- Transfer a domain
- Getting a domain transfer
- Cancel an in progress domain transfer

### CODE REVIEW

**Review by aeden — APPROVED** [review]

**Review by san983 — APPROVED** [review]

**Comment by 1atAlcone** [comment]
- [ ]

File: curl/cancel_domain_transfer.sh
Comment (1atAlcone) [inline curl/cancel_domain_transfer.sh:1]: - [ ]

File: node/domain_transfers.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+// Usage: TOKEN=token node domain_transfers.js example.com 42
+// where 42 is the domain transfer id
+
```
Comment (1atAlcone) [inline node/domain_transfers.js:5]: -

File: node/transfer_domain.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+// Usage: TOKEN=token node transfer_domain.js example.com auth_code:test registrant_id:42
+
+var client = require('dnsimple')({
+  baseUrl: 'https://api.sandbox.dnsimple.com',
+  accessToken: process.env.TOKEN,
+});
```
Comment (1atAlcone) [inline node/transfer_domain.js:8]: 0 lol

File: ruby/cancel_domain_transfer.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+require 'pp'
+require 'dnsimple'
+require_relative 'token'
+
```
Comment (1atAlcone) [inline ruby/cancel_domain_transfer.rb:5]: IoS seo0

**Review by 1atAlcone — COMMENTED** [review]
Haha

**Pull request merged**